Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:3.2.0-rc1-alpine3.17
+
+RUN apk add --update make
+RUN adduser -D userRuby
+
+USER userRuby
+
+WORKDIR /home/userRuby
+
+COPY --chown=userRuby Gemfile Gemfile.lock ./
+
+RUN gem install bundler:2.2.33
+RUN bundle install
+
+WORKDIR /app/test
+
+CMD ["make", "test"]
```

Comment:
¿Por qué usas CMD y no ENTRYPOINT?

Reply:
También he seguido buenas prácticas, en la guía que encontré es lo que se usa

Reply:
Guía de buenas prácticas ¿para qué?

Reply:
Para construir imágenes de Docker para Ruby: https://lipanski.com/posts/dockerfile-ruby-best-practices

No lo dice explícitamente, pero es lo que usa en todos los ejemplos, y dice también que utilice ese formato. Si por otro motivo es mejor usar ENTRYPOINT lo cambio, pero me gustaría saber por qué.

Reply:
No te estoy diciendo que uses ENTRYPOINT. Te estoy diciendo que lo justifiques y que aprendas la diferencia entre uno y otro y sus casos de uso. 
En cuanto a lo que enlazas, dice claramente Ruby *application*. Esto es una imagen *para test*. Diferente caso de uso, diferentes buenas prácticas. Aunque muchas cosas serán aplicables a los dos, otras pueden contradecir tanto lo necesario para test como lo necesario para que yo evalúe si habéis alcanzado vuestros objetivos de aprendizaje. En todo caso, creo que no la has enlazado, así que conviene que lo hagas.